.github/workflows/plugins_test.yml

```diff
@@ -0,0 +1,64 @@
+name: Plugins test
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - repo: katello
+            org: laviro
+            ref: feat/foreman_js_link
+          - repo: foreman-tasks
+            org: laviro
+            ref: feat/foreman_js_link
```

Comment:
added also to foreman-tasks the `foreman-js:link` script: https://github.com/theforeman/foreman-tasks/pull/568

Reply:
got merged :+1: